### DISCUSSION TITLE
[2021/08/18]KubeSphere v3.2 meeting

### DISCUSSION THREAD

**Discussion by zryfish** [kubesphere/kubesphere#4152] [Community Meeting]
## Meeting
**Time** : 2021/08/18 14:00 
**Location**:: Online
**Topic**: KubeSphere 3.2 Schedule
**Tecent Meeting**:  105985201   https://meeting.tencent.com/dm/2jwPE26qw1KN 

### sig/multicluster @yuswift 
* #4157 
* #3246 
* #3898 
* #3008
* #3970

### sig/storage
*  Manage storage capabilities in console. [#4075](https://github.com/kubesphere/kubesphere/issues/4075)[Need Merage]
*  Manage pv in console. [#4074](https://github.com/kubesphere/kubesphere/issues/4074)[Under developement]

### sig/observability
* Notification filter [#4087](https://github.com/kubesphere/kubesphere/issues/4087) [WIP]
* Notificaiton setting validation [#4087](https://github.com/kubesphere/kubesphere/issues/4087) [WIP]
* containerd/CRI-O log support [#4081](https://github.com/kubesphere/kubesphere/issues/4081) [Done]
* FluentBit Operator & FluentBit upgrade [#4081](https://github.com/kubesphere/kubesphere/issues/4081) [WIP]
* Upgrade kube-auditing CRD to v1[4148](https://github.com/kubesphere/kubesphere/issues/4148) [WIP]
* Set hostpath to readonly for relevant components [4149](https://github.com/kubesphere/kubesphere/issues/4149) [WIP]
* Upgrade admission webhook from v1beta1 to v1 [4150](https://github.com/kubesphere/kubesphere/issues/4150) [Done]

### sig/edge
* Upgrade KubeEdge to v1.7.2 [4086](https://github.com/kubesphere/kubesphere/issues/4086) [WIP]
* Upgrade CRD to v1 [4086](https://github.com/kubesphere/kubesphere/issues/4086) [WIP]

### sig/console
* Specify the annotation of the pod in create workload modal. [#2115](https://github.com/kubesphere/console/issues/2115)[Need Merage]
* Request to fix the potential secret leak. [#2131](https://github.com/kubesphere/console/issues/2131)[Need Merage]
* Support to display multiple ip's of pods.[#2116](https://github.com/kubesphere/console/issues/2116)[Done]
* Missing gitlab option in 'Select Code Repository'. [#2094](https://github.com/kubesphere/console/issues/2094)[Done]
* It would be better that there are "+" and "-" to show the number of replicas is clickable.[#2135](https://github.com/kubesphere/console/issues/2135)[Done]
* How to specify the sub-path of console [#1884](https://github.com/kubesphere/console/issues/1884)[Need to Discuss]
* Enhance: provide a secret setup link when setting env of a container.[#88](https://github.com/kubesphere/console/issues/88)[Need to Discuss]
* GPU workload support [4083](https://github.com/kubesphere/kubesphere/issues/4083) [WIP]

### sig/api
* Refactor KubeSphere gateway with CRD. [#3055](https://github.com/kubesphere/kubesphere/issues/3055) [WIP]
* Upgrade k8s package version. [#4141](https://github.com/kubesphere/kubesphere/pull/4141)

### sig/auth
* https://github.com/kubesphere/kubesphere/issues/3378 [WIP]

### sig/security
* https://github.com/kubesphere/kubesphere/issues/4139

### sig/apps
* Upgrade kustomize to v4.2.0. [#4115](https://github.com/kubesphere/console/issues/4115)[Done]
* Cleanup apps when deleting workspace. [#3915](https://github.com/kubesphere/console/issues/3915)[Under developement]

### sig/devops

* [Closed](https://github.com/kubesphere/ks-devops/issues?q=is%3Aissue+milestone%3Av3.2+is%3Aclosed)
* [WIP](https://github.com/kubesphere/ks-devops/issues?q=is%3Aissue+milestone%3Av3.2+is%3Aopen)

### sig/installation
* Set read-only permissions for hostpath volume. [#1628](https://github.com/kubesphere/ks-installer/issues/1628)[Done]
* Support custom components' resources. [#1620](https://github.com/kubesphere/ks-installer/issues/1620)[Under developement]
* Support custom components' scheduling. [#1636](https://github.com/kubesphere/ks-installer/issues/1636)[Under developement]
* DCGM exporter for GPU monitoring [4082](https://github.com/kubesphere/kubesphere/issues/4082) [Done]

### sig/docs
* UI text optimization plan
* Documentation optimization plan

**Comment by FeynmanZhou** [comment]
@zryfish Will this meeting be weekly or biweekly in the future?

**Reply by zryfish** [reply]
Yes, biweekly.

**Comment by FeynmanZhou** [comment]
We should add the new features of v3.2 into the Roadmap. This topic could be added to the meeting agenda.

**Reply by liangyuanpeng** [reply]
> We should add the new features of v3.2 into the Roadmap. This topic could be added to the meeting agenda.

Absolutely right, I try to found v3.2 from roadmap and got nohting.   

Actually, I spent a lot of time searching for roadmap. It's greate If we can make the Roadmap easier to find.

**Comment by FeynmanZhou** [comment]
@Patrick-LuoYu Are there any updates that need to be added to the agenda from @kubesphere/sig-docs ?
